golang-github-osbuild-composer.spec

```diff
@@ -68,9 +68,17 @@ export GOFLAGS=-mod=vendor
 %gobuild -o _bin/osbuild-worker %{goipath}/cmd/osbuild-worker
 %gobuild -o _bin/osbuild-tests %{goipath}/cmd/osbuild-tests
 %gobuild -o _bin/osbuild-weldr-tests %{goipath}/cmd/osbuild-weldr-tests
-%gobuild -o _bin/osbuild-dnf-json-tests %{goipath}/cmd/osbuild-dnf-json-tests
 %gobuild -o _bin/osbuild-image-tests %{goipath}/cmd/osbuild-image-tests
 
+# Build test binaries with `go test -c`, so that they can take advantage of
```

Comment:
Have you tried this on RHEL? Also, I'm a bit worried what happens when `%gobuild` definition changes in future Fedora/RHEL versions.

Reply:
Yeah, it is a bit hacky indeed. I've changed it like @teg suggested. Building it in such a way that still works without the internal variables. I do hope it works everywhere: I tried Fedora 31, but didn't have time for RHEL yet. Will try that in the morning.

Reply:
@ondrejbudai noted below that he checked that this works.